### PR TITLE
Misprints

### PR DESCRIPTION
**Pull request by outcoldman**

### CODE REVIEW

**Review by lephino — APPROVED** [review]

**Review by bb03 — APPROVED** [review]

**Comment by lephino** [comment]
@outcoldman after you have 2 approvals you can merge, and please close out any issues associated with this pr.  If you're finished with the branch, fill free to trash that as well.

**Review by nwang92 — APPROVED** [review]

**Comment by outcoldman** [comment]
Don't believe I have a right to merge

screen shot 2018-10-08 at 2 53 26 pm

**Comment by lephino** [comment]
K, my mistake merging it in.

**Pull request merged**
